testing/test_aux.out

```diff
@@ -5,6 +5,9 @@ Mudacl exit code 0
 Validator exit code 0
 Lint checks
 cmd/inbuild exit code 0
+Running ./testing/test_bacext.sh
```

Comment:
How is this added if your change is to test_bacext.sh?  This file (test_aux.out) seems correct, but that other one shouldn't exist, as per my other comment.